subworkflows/local/metagenomics.nf

```diff
@@ -0,0 +1,59 @@
+include { METAGENOMICS_COMPLEXITYFILTER } from './metagenomics_complexityfilter'
+include { METAGENOMICS_PROFILING        } from './metagenomics_profiling'
+include { METAGENOMICS_POSTPROCESSING   } from './metagenomics_postprocessing'
+
+workflow METAGENOMICS {
+    take: ch_bamfiltered_for_metagenomics
+    take: ch_database
+    take: ch_tax_list
+    take: ch_ncbi_dir
+
+    main:
+    // Define channels
+    ch_multiqc_files                = Channel.empty()
+    ch_versions                     = Channel.empty()
+
+    // Important Note:
+    // Due to the bamfilter submodule
+    // The single_end parameter in the meta is always true!
```

Comment:
Ooof, can you double check this in latest `-r dev`? It may not be any more

Reply:
still is, will be continued in separate PR